doc/Makefile

```diff
@@ -5,10 +5,10 @@ include Makefile.sphinx
 
 CHPL2RST     = ../util/run-in-venv.bash ./util/chpl2rst.py
 
-CHPL_DOC_PRIMERS_DIR = primers/primers
+CHPL_DOC_PRIMERS_DIR = $(SOURCEDIR)/primers/primers
```

Comment:
Actually, I take that back -- since this PR updates so many references to `primers/primers` already, it would be easier to do all the updates at once, and include the change in this PR.